test/bdd/features/query.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Querying assets on the DKG
+
+    Scenario: Querying the assertion by keyword on the DKG
+        Given query request is sent for the keyword
```

Comment:
This line should look like this
Given i send query request on node 1 for the keyword ....
Then after this always goes:
Then I wait for last query to be finalized
Then you should write a step for validation:
And last query finished with status: COMPLETED
And last query returned valid result